readme.txt

```diff
@@ -2,10 +2,10 @@
 Contributors:      jakemgold, rcbth, 10up, thinkoomph, tlovett1
 Donate link:       https://10up.com/plugins/restricted-site-access-wordpress/
 Tags:              privacy, restricted, restrict, privacy, limited, permissions, security, block
-Requires at least: 4.6
+Requires at least: 5.0
 Tested up to:      5.9
 Stable tag:        7.3.0
-Requires PHP:      
+Requires PHP:      7.3
```

Comment:
Looks like previously we weren't setting a minimum number here. While I like setting this as high as possible, is there any reason we couldn't set this to 7.0? I guess my concern here is by setting this to 7.3, anyone using a PHP version lower than that won't be able to install (or at least I think WordPress enforces this). And if we're not doing anything at a code-level that wouldn't work on 7.0+, wondering if we just 7.0 as our requirement?

Reply:
@Sidsector9 thoughts on Darin's comment here?  I'd love to get this rolled into the 7.3.1 release we're targeting for next week.

Reply:
@dkotter that was the requirement for PHPUnit 9.4.4. Even if we leave PHPUnit at 7.5, the minimum PHP version required is still 7.1.

The production build of the plugin requires minimum PHP 5.6 and the development version requires minimum PHP 7.3 if you need to run PHPUnit tests.

I have reverted the minimum PHP version for the plugin to 5.6.
However, both PHPCompatibility and PHPUnit actions run in an environment with PHP 7.3 as PHPUnit 9.4.4 is locked to it.